packages/web-components/fast-foundation/src/data-grid/data-grid-cell.ts

```diff
@@ -235,15 +268,38 @@ export class FASTDataGridCell extends FASTElement {
                 break;
 
             case keyEscape:
-                if (
-                    this.contains(document.activeElement) &&
-                    document.activeElement !== this
-                ) {
+                if (this.contains(rootActiveElement) && rootActiveElement !== this) {
                     this.focus();
                     e.preventDefault();
                 }
                 break;
+
+            // stop any unhandled grid nav events that may bubble from the cell
+            // when internal navigation is active.
+            // note: preventDefault would also block arrow keys in input elements
+            case keyArrowDown:
+            case keyArrowLeft:
+            case keyArrowRight:
+            case keyArrowUp:
+            case keyEnd:
+            case keyHome:
+            case keyPageDown:
+            case keyPageUp:
```

Comment:
could this just be `default`?

Reply:
No, we don't want to catch all the keys, only the ones that could cause the parent grid to move focus around.